docs/guide/installation/local-install.mdx

```diff
@@ -81,4 +81,19 @@ Then, in your terminal (from the project folder):
 botfront set-project 3PEj5txDenAwG5brc # or whatever your project ID is
 ```
  
-Congrats! Your project is now ready!
\ No newline at end of file
+Congrats! Your project is now ready!
+
+
+## Secure rasa access (optional)
+
+You might want to secure the communication between botfront and rasa-for-botfront.
+
+- In the instance settings there is a token, you can edit it or generate new ones as you feel.
+![](../../images/auth_token_instance_settings.png)
+
+- Add this token in env part of the botfront.yml under the key auth_token
```

Comment:
don't show an image, just yaml with the relative path from project root as a caption